salt/pythontest/config/vsftpd.conf

```diff
@@ -0,0 +1,45 @@
+ftpd_banner=Welcome to pythontest FTP server
+
+# Run vsftp as a standalone server
+listen=YES
+
+# Allow anonymous FTP. (This is what the test suite uses)
+anonymous_enable=YES
+
+# The only local users are the python infra team and they
+# do not use ftp for administration
+local_enable=NO
+
+# This option prevents any writing ftp commands from being issued
+write_enable=NO
+
+# Activate directory messages - messages given to remote users when they
+# go into a certain directory.
+dirmessage_enable=YES
+
+# If enabled, vsftpd will display directory listings with the time
+# in  your  local  time  zone.  The default is to display GMT. The
+# times returned by the MDTM FTP command are also affected by this
+# option.
+use_localtime=YES
+
+# Activate logging of uploads/downloads.
+xferlog_enable=YES
+
+# Make sure PORT transfer connections originate from port 20 (ftp-data).
+connect_from_port_20=YES
+# These ports must be opened in the firewall to enable data transfer
+pasv_enable=YES
+pasv_addr_resolve=YES
+pasv_address=www.pythontest.net
+pasv_max_port=10100
+pasv_min_port=10090
+
+# This option should be the name of a directory which is empty.  Also, the
+# directory should not be writable by the ftp user. This directory is used
+# as a secure chroot() jail at times vsftpd does not require filesystem
+# access.
+secure_chroot_dir=/var/run/vsftpd/empty
+
+# This string is the name of the PAM service vsftpd will use.
+pam_service_name=vsftpd
```

Comment:
For what its worth I think PAM won't be used at all since we set `local_enable=NO` so it doesn't do any user authentication but I don't mind disabling this just in case.